src/inc/corinfo.h

```diff
@@ -1312,6 +1312,7 @@ enum CORINFO_RUNTIME_LOOKUP_KIND
 struct CORINFO_LOOKUP_KIND
 {
     bool                        needsRuntimeLookup;
+    WORD                        runtimeLookupFlags;
```

Comment:
There should be comment about this - that this is just for internal VM/ZAP communication, not to be used by the JIT.